scripts/hid_configurator/modules/dfu.py

```diff
@@ -357,7 +379,12 @@ def fwinfo(dev):
 
 
 def fwreboot(dev):
-    success, fetched_data = dev.config_get('dfu', 'reboot')
+    dfu_module_name = dev.get_complete_module_name('dfu')
+    if dfu_module_name:
+        success, fetched_data = dev.config_get(dfu_module_name , 'reboot')
+    else:
+        print('Module DFU not found')
+        return None
```

Comment:
Shouldn't we return `False, False` as `success, rebooted` pair? To keep values returned by the function consistent.